src/NuGet.Core/NuGet.Commands/VerifyCommand/VerifyCommandRunner.cs

```diff
@@ -92,21 +92,30 @@ private async Task<int> VerifySignatureForPackageAsync(string packagePath, ILogg
 
                 var packageIdentity = packageReader.GetIdentity();
 
-                logger.LogInformation(Environment.NewLine + string.Format(CultureInfo.CurrentCulture,
+                logger.LogMinimal(Environment.NewLine + string.Format(CultureInfo.CurrentCulture,
                     Strings.VerifyCommand_VerifyingPackage,
                     packageIdentity.ToString()));
                 logger.LogInformation($"{packagePath}{Environment.NewLine}");
 
                 var logMessages = verificationResult.Results.SelectMany(p => p.Issues).ToList();
-                await logger.LogMessagesAsync(logMessages);
+
+                //log issues first
```

Comment:
Is this comment correct? You're logging non-issues first.

Reply:
yes Erick, the comment is correct. The idea is to log all the issues whose level is `not warning or error` first. If you think it needs to be tweaked, I will create a follow up PR because this PR has passed all checks.

Reply:
Ok. I thought `issue` means something has problem such as warnings and errors.
I believe correct wording would be '//log non-issues first'.
Of you can address in another PR.

Reply:
I can address in this comment in this PR, but my only concern is it may take 2 or 3 attempts for a green build.

Reply:
Ok. I approved this PR.